io.github.limo_app.limo.json

```diff
@@ -0,0 +1,351 @@
+        "-DBUILD_SHARED_LIBS=OFF",
+        "-DBUILD_OBJECT_LIBS=OFF"
+      ],
+      "sources":
+      [
+        {
+          "type": "git",
+          "url": "https://github.com/open-source-parsers/jsoncpp.git",
+          "tag": "1.9.5"
+        }
+      ]
+    },
+    {
+      "name": "pugixml",
+      "buildsystem": "cmake-ninja",
+      "config-opts":
+      [
+        "-DCMAKE_BUILD_TYPE=Release"
+      ],
+      "sources":
+      [
+        {
+          "type": "git",
+          "url": "https://github.com/zeux/pugixml.git",
+          "tag": "v1.14"
+        }
+      ]
+    },
+    {
+      "name": "zlib",
+      "buildsystem": "cmake-ninja",
+      "config-opts":
+      [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DBUILD_SHARED_LIBS=OFF",
+        "-DBUILD_STATIC_LIBS=ON"
+      ],
+      "sources":
+      [
+        {
+          "type": "git",
+          "url": "https://github.com/madler/zlib.git",
+          "tag": "v1.3.1"
+        }
+      ],
+      "cleanup":
+      [
+        "/lib/libz.so*"
+      ]
+    },
+    {
+      "name": "curl",
+      "buildsystem": "cmake-ninja",
+      "config-opts":
+      [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DBUILD_SHARED_LIBS=OFF",
+        "-DBUILD_STATIC_LIBS=ON"
+      ],
+      "sources":
+      [
+        {
+          "type": "git",
+          "url": "https://github.com/curl/curl.git",
+          "tag": "curl-8_9_1"
+        }
+      ]
+    },
+    {
+      "name": "cpr",
+      "buildsystem": "cmake-ninja",
+      "config-opts":
+      [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DCPR_USE_SYSTEM_CURL=ON",
+        "-DBUILD_SHARED_LIBS=OFF"
+      ],
+      "sources":
+      [
+        {
+          "type": "git",
+          "url": "https://github.com/libcpr/cpr.git",
+          "tag": "1.10.5"
+        }
+      ]
+    },
+    {
+      "name": "libtbb",
+      "buildsystem": "cmake-ninja",
+      "config-opts":
+      [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DTBB_TEST=OFF",
+        "-DBUILD_SHARED_LIBS=OFF",
+        "-DTBB_STRICT=OFF"
+      ],
+      "sources":
+      [
+        {
+          "type": "git",
+          "url": "https://github.com/oneapi-src/oneTBB.git",
+          "tag": "v2021.13.0"
+        }
+      ]
+    },
+    {
+      "name": "boost",
+      "buildsystem": "cmake-ninja",
+      "config-opts":
+      [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DBOOST_INCLUDE_LIBRARIES=algorithm;crc;graph"
+      ],
+
+      "sources":
+      [
+        {
+          "type": "archive",
+          "url": "https://github.com/boostorg/boost/releases/download/boost-1.85.0/boost-1.85.0-cmake.tar.xz",
+          "sha256": "0a9cc56ceae46986f5f4d43fe0311d90cf6d2fa9028258a95cab49ffdacf92ad"
+        }
+      ]
+    },
+    {
+      "name": "icu",
+      "buildsystem": "simple",
+      "build-commands":
+      [
+        "cd source && ./configure --prefix /app/local --enable-static=yes --enable-shared=no",
```

Comment:
this should be `buildsystem: autotools` anyway. You are probably looking at `subdir: source`. But see above: it's in the runtime.